web/keyApi.go

```diff
@@ -10,6 +10,7 @@ import (
 var (
 	keyAllApi    = regexp.MustCompile(`^\/api\/v1\/apikey[\/]*$`)
 	keySingleApi = regexp.MustCompile(`^\/api\/v1\/apikey\/(.*)$`)
+	keyListApi   = regexp.MustCompile(`^\/api\/v1\/apikeys[\/]*$`)
```

Comment:
We don't need a new route for this. We can reuse one of the existing ones. Just use a different http verb GET